### DISCUSSION TITLE
[dragMode] possibility to select the side of the screen to base on

### DISCUSSION THREAD

**Discussion by voleep** [GrapesJS/grapesjs#4757] [General]
@artf maybe could have a way to let the user to select the side position where dragMode uses as base side to calculate the position of the element?

For example, by default dragMode: 'absolute' moves an element in the canvas based on the **top** and **left** side and it's a pain if we want to set an element to be fixed on other sides of screen.

Also having the possibility to change px to % should take the GrapesJS to the next level

**Comment by artf** [comment]
We have an issue [here](https://github.com/artf/grapesjs/issues/3770) tracking all the problems with the dragMode but unfortunately I didn't find time yet to spend on that feature so far.
